docs/how-to-guides/meson-args.rst

```diff
@@ -85,33 +62,37 @@ To set this option temporarily at build-time:
 
         .. code-block:: console
 
-			 $ python -m build -Csetup-args="--default-library=static" .
+           $ python -m build -Csetup-args="--default-library=static" .
 
     .. tab-item:: pip
         :sync: key_pip
 
         .. code-block:: console
 
-			 $ python -m pip --config-settings=setup-args="--default-library=static" .
+	   $ python -m pip wheel . --config-settings=setup-args="--default-library=static" .
 
 
-2) Use Meson install_tags for selective installs
+Select the build targets to include in the wheel
 ------------------------------------------------
 
-Meson install_tags can be used (since ``meson-python`` >= 0.13) to select which
-targets are installed into the binary wheels. This example causes meson-python
-to only install targets tagged with ``runtime`` or ``python-runtime``) into the
-binary wheel (ignoring e.g. C++ headers).
+It is possible to include in the Python wheel only a subset of the
+installable files using Meson `installation tags`_ via the ``meson
+install``'s ``--tags`` command line option. When ``--tags`` is
+specified, only files that have been tagged with one of the tags are
+going to be installed. Meson sets predefined tags on some
+files. Custom installation tag can be set using the ``install_tag``
+keyword argument passed to the target definition function.  In this
+example only targets tagged with ``runtime`` or ``python-runtime`` are
+included in the Python wheel.
```

Comment:
I'd make a new paragraph here, also suggested rewording.

I'd skip the large tags explanation, though, and simply point the user to the Meson docs. This is supposed to just show you what you can do, not to be a full tutorial (we can make it a full tutorial, though).

typo: "Custom installation tag can" > "Custom installation tags can"

```suggestion
install``'s ``--tags`` command line option. When ``--tags`` is specified,
only files that have one of the specified tags are going to be installed.

Meson sets predefined tags on certain files. Custom installation tags can
be set using the ``install_tag`` keyword argument passed to the target
definition function.  In this example only targets tagged with ``runtime``
or ``python-runtime`` are included in the Python wheel.
```

Reply:
This text was in the document before. I just edited it be readable.